pkg/chartutil/jsonschema.go

```diff
@@ -91,3 +99,51 @@ func ValidateAgainstSingleSchema(values Values, schemaJSON []byte) (reterr error
 
 	return nil
 }
+
+// keep the old behaviour for empty $schema or the one that defined in
+// https://github.com/xeipuuv/gojsonschema/blob/v1.2.0/draft.go#L46-L62
+func validateUsingNewValidator(valuesJSON, schemaJSON []byte) (bool, error) {
+	var partialSchema struct {
+		Schema string `json:"$schema"`
+	}
+	_ = json.Unmarshal(schemaJSON, &partialSchema)
+	if partialSchema.Schema == "" {
+		return false, nil
+	}
+
+	url, err := url.Parse(partialSchema.Schema)
+	if err != nil {
+		return false, nil
+	}
+	if url.Host == "json-schema.org" {
+		switch url.EscapedPath() {
+		case
+			"/draft-04/schema",
+			"/draft-06/schema",
+			"/draft-07/schema":
+			return false, nil
```

Comment:
Do we want to support `2019-09`? Given that it has been superseded by `2020-12`. And since Helm never supported it, there should be no reason someone would prefer it over `2020-12`?

Reply:
I have no preference about this, I only follow https://github.com/xeipuuv/gojsonschema/blob/v1.2.0/draft.go#L46-L62

Reply:
My vote would be to not support `2019-09`. I'm not seeing any benefit for doing so. If a reason occurs in the future, support could be included then.

The downsides to supporting included increased support burden, greater potential for bugs, etc.

Reply:
I'm not sure how I can remove support for `2019-09` here, this switch case only chooses if the code should use the old library or the new library.

are you suggesting that `2019-09` should be validated using old library `github.com/xeipuuv/gojsonschema`, which clearly does not support at all

Reply:
> are you suggesting that 2019-09 should be validated using old library github.com/xeipuuv/gojsonschema, which clearly does not support at all

effectively "yes". In that by not supporting `2019-09` here and returning to the old library: the old behavior is retained. ie. the same behavior that `"$schema": "http://json-schema.org/2019-09/schema#"` would invoke prior to this PR (Which presumably will be to error)

Reply:
support to 2019 schema is removed, it now explicitly only support 2020